build/build-options.js

```diff
@@ -0,0 +1,106 @@
+require("dotenv").config();
+
+const {
+  FLAW_LEVELS,
+  DEFAULT_FLAW_LEVELS,
+  VALID_FLAW_CHECKS,
+  FILES,
+  FOLDERSEARCH,
+  NO_PROGRESSBAR,
+} = require("./constants");
+
+module.exports = {
```

Comment:
nit: I'd vote for moving it down to not rely on function hoisting

Reply:
Got it.